packages/thirdweb/src/react/web/wallets/shared/ConnectWalletSocialOptions.tsx

```diff
@@ -91,6 +91,7 @@ export const ConnectWalletSocialOptions = (
     facebook: locale.signInWithFacebook,
     apple: locale.signInWithApple,
     discord: locale.signInWithDiscord,
+    farcaster: "Farcaster",
```

Comment:
I guess can't localize this anyways?

Reply:
Yeah the locales are just the names anyway, feels unnecessary